benchmarks/hollow_sphere/hollow_sphere.cc

```diff
@@ -0,0 +1,702 @@
+#include <aspect/material_model/simple.h>
+#include <aspect/boundary_velocity/interface.h>
+#include <aspect/simulator_access.h>
+#include <aspect/global.h>
+#include <aspect/gravity_model/interface.h>
+#include <aspect/utilities.h>
+
+#include <deal.II/dofs/dof_tools.h>
+#include <deal.II/numerics/data_out.h>
+#include <deal.II/base/quadrature_lib.h>
+#include <deal.II/base/function_lib.h>
+#include <deal.II/numerics/error_estimator.h>
+#include <deal.II/numerics/vector_tools.h>
+
+
+
+namespace aspect
+{
+  /**
+   * This is the "Hollowsphere" benchmark defined in the following paper:
+   */
```

Comment:
yes, the low resolution case (768 elements) runs quickly and yes I will add a page in the manual and a test.